src/picasso-definition/components/mini-chart/background-window.js

```diff
@@ -0,0 +1,26 @@
+import KEYS from '../../../constants/keys';
+import NUMBERS from '../../../constants/numbers';
+
+export default function createMiniChartBackgroundWindow(chartModel) {
+  const ratio = NUMBERS.MINI_CHART.RATIO;
+  const padding = NUMBERS.MINI_CHART.PADDING; // Padding from the bottom right corner
```

Comment:
I would probably do `const { RATIO, PADDING } = NUMBERS.MINI_CHART` to make it clearer where they are used that they are constants.